conductor-modules/arch-components-lifecycle/src/main/java/com/bluelinelabs/conductor/archlifecycle/ViewModelController.java

```diff
@@ -0,0 +1,17 @@
+package com.bluelinelabs.conductor.archlifecycle;
+
+import android.arch.lifecycle.ViewModelProvider;
+import android.arch.lifecycle.ViewModelStore;
+
+public abstract class ViewModelController extends LifecycleController {
+
+    private ViewModelStore viewModelStore = new ViewModelStore();
```

Comment:
I think `viewModelStore.clear()` should be called on `Controller` `onDestroy`

Reply:
Agree, as the docs say: ```If an owner of this ViewModelStore is destroyed and is not going to be recreated, then it should call clear() on this ViewModelStore, so ViewModels would be notified that they are no longer used.``` https://developer.android.com/reference/android/arch/lifecycle/ViewModelStore.html#clear() I'd look to add that to the ViewModelController onDestroy method.